docs/development/workflow/get_devel_version.rst

```diff
@@ -222,30 +222,43 @@ of `Astropy`_, for now this straightforward way will work (if you want to jump
 ahead to the more sophisticated method look at :ref:`virtual_envs`).
 
 .. note::
-    There are a couple of circumstances in which this quick method of
+    There are some circumstances in which this quick method of
     activating your copy of `Astropy`_ will NOT work and you need to go
-    straight to using a virtual python environment:
+    straight to using a virtual python environment, for example:
 
-    + You use Python 3.
     + You want to work on C or Cython code in `Astropy`_.
 
+If you have decided to use the recommended "activation" method with pip, 
+please note the following: Now is the time to check that you have all 
+the required dependencies installed on your machine to make sure 
+the proposed pip install would be a successful one. Namely, as 
+you are effectively building from source, please check that you have both "cython"
+as well as "jinja2" installed using pip. (Note: Using conda to install these two 
+dependencies would normally not work and would result in error(s) thrown.) 
+Please note that by pip we really mean the python 3 version of pip. 
+If you have any problem with different versions of pip installed,
+try aliasing to resolve the issue. If you are unsure about which pip version you 
+are using, try the command ``which python`` on the terminal. 
 In the directory where your copy of `Astropy`_ is type::
 
     pip install -e .[all,test]
 
 Several pages of output will follow the first time you do this; this wouldn't
 be a bad time to get a fresh cup of coffee. At the end of it you should see
-something like  ``Finished processing dependencies for astropy==0.3.dev6272``.
+something like  ``Finished processing dependencies for astropy==3.2.dev6272``.
 
 To make sure it has been activated **change to a different directory outside of
 the astropy distribution** and try this in python::
 
     >>> import astropy
     >>> astropy.__version__  # doctest: +SKIP
-    '0.3.dev6272'
+    '3.2.dev6272'
 
 The actual version number will be different than in this example, but it
-should have dev in the name.
+should have ``'dev'`` in the name.
+
+There are other ways to perform the same "activation"/installation task, 
```

Comment:
I think we should just recommend one way, so would remove this.  Or move all details to footnotes at the end.

Reply:
Yup, removed...